src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -4597,52 +4258,54 @@ namespace HVACMultiSpeedHeatPump {
 
         QCoilActual = 0.0;
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:4600]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const